SPECS/libxcrypt/libxcrypt.spec

```diff
@@ -408,13 +377,8 @@ ln -s %{_libdir}/libcrypt-%{glibcversion}.so %{_libdir}/libcrypt.so.1
 %files devel
 %doc %{_pkgdocdir}/ChangeLog
 %doc %{_pkgdocdir}/TODO
-%if %{with override_glibc}
-%exclude %{_libdir}/libcrypt.so
-%exclude %{_includedir}/crypt.h
-%else
 %{_libdir}/libcrypt.so
```

Comment:
this will mean that we'll also need a glibc-libcrypt-devel package that conflicts with the libxcrypt-devel package, right?